lib/final-api/endpoint/ddtf.rb

```diff
@@ -64,7 +64,7 @@ def registered(app)
 
             halt 400, enqueue_data.errors.to_json unless enqueue_data.valid?
 
-            config = DdtfHelpers.build_config(payload, tsd)
+            config = DdtfHelpers.build_config(payload, JSON.parse(enqueue_data.enqueue_data['TsdContent']))
```

Comment:
I rather implement this by adding tsd_content method to TsdUtils that reads info from enqueue data. If you refactor TsdUtil consider change of propagating build_id into TsdUtils result.